client/my-sites/customer-home/cards/tasks/site-setup-list/style.scss

```diff
@@ -65,6 +65,20 @@
 				border-bottom: none;
 			}
 		}
+
+		.site-setup-list__email {
+			display: none;
+			@include break-mobile {
+				color: var( --studio-gray-20 );
+				display: block;
+			}
+		}
+
+		.site-setup-list__emailbold {
```

Comment:
I have changed the class names to `.site-setup-list__nav-item-email` and `.site-setup-list__task-description-email` so that it shows the relationship to the parent component [as per documentation](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#classes). Let me know if that is alright

Reply:
Those sound good to me, thanks 👍